tests/shared/models/flash-state.spec.js

```diff
@@ -271,7 +272,9 @@ describe('Model: flashState', function () {
         flashState.setFlashingFlag()
         flashState.setProgressState(state)
         const currentFlashState = flashState.getFlashState()
-        m.chai.expect(currentFlashState).to.deep.equal(state)
+        m.chai.expect(currentFlashState).to.deep.equal(_.assign(state, {
+          speed: `${state.speed} B`
+        }))
```

Comment:
Could we also add some tests where we check other speed values get the correct suffix appended? (e.g. KB/s, MB/s)

Reply:
I don't think we need to be testing `pretty-bytes` in Etcher... ;)

Reply:
The suggestion here wasn't to test `pretty-bytes` itself, but to test that our code is correctly _using_ `pretty-bytes` the way we expect it to :stuck_out_tongue_winking_eye:

Reply:
I still think that it'd be nice to have at least one test that checks a value other than '0 B/s' (i.e. testing setProgressState with a state.speed other than 0).